### PR TITLE
Adding README

### PR DESCRIPTION
**Pull request by timidri**
Adding README file to review. The README file is referring to REFERENCE.md which is auto-generated from in-code comments.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,416 @@
+# Puppet module for a PowerStore Storage Appliance
```
Comment (AronAtDell) [inline README.md:1]: This should say "Puppet module for Dell EMC PowerStore."

File: README.md [hunk 2]
```diff
@@ -0,0 +1,416 @@
...
+
+#### Table of Contents
+
+- [Puppet module for a PowerStore Storage Appliance](#puppet-module-for-a-powerstore-storage-appliance)
```
Comment (AronAtDell) [inline README.md:5]: Same comment as above. This should say "Puppet module for Dell EMC PowerStore."

Comment (AronAtDell) [reply README.md:5]: I would recommend that you replace all instance of "PowerStore" with "Dell EMC PowerStore"

File: README.md [hunk 3]
```diff
@@ -0,0 +1,416 @@
...
+          user: admin
+          password: My$ecret!
+          remote-transport: powerstore
+   ```
+
+## Usage
+
+### Using Tasks
+
+#### Introduction to PowerStore tasks
```
Comment (AronAtDell) [inline README.md:89]: Should say "Introduction to Dell EMC PowerStore tasks."

File: README.md [hunk 4]
```diff
@@ -0,0 +1,416 @@
...
+
+```
+pdk bundle exec rake strings:generate:reference
+```
+
+## Contributors
+
+## Contact
+
+... forthcoming ...
```
Comment (AronAtDell) [inline README.md:411]: Do we know these at this point or there would be another PR for this?

File: README.md [hunk 5]
```diff
@@ -0,0 +1,416 @@
...
+### Generating REFERENCE.md
+
+To (re-)generate the REFERENCE.md file which documents the available types, tasks, functions and plans, run:
+
+```
+pdk bundle exec rake strings:generate:reference
+```
+
+## Contributors
+
```
Comment (AronAtDell) [inline README.md:408]: Will review this once available. Would this just point to a how to document on contributing?

File: README.md (hunk 1 above)
Comment (timidri) [reply README.md:1]: Done.

File: README.md (hunk 2 above)
Comment (timidri) [reply README.md:5]: Done.

File: README.md (hunk 3 above)
Comment (timidri) [reply README.md:89]: Done

**Review by AronAtDell — APPROVED** [review]
Changes look good. Thanks!

**Review by ameer-jabbar — APPROVED** [review]
